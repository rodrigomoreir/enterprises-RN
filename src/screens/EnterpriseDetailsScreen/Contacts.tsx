import React from 'react'

import styled from 'styled-components/native'

interface Props {
  meansOfContact: string
  contact: string
}

const Contacts = ({ meansOfContact, contact }: Props) => {
  return (
    <StyledContainer>
      <StyledTitle>{meansOfContact}</StyledTitle>
      <StyledDescription>{contact}</StyledDescription>
    </StyledContainer>
  )
}

const StyledContainer = styled.View`
  flex-direction: row;
  align-items: flex-end;
`
const StyledTitle = styled.Text`
  margin-top: 10px;
  font-size: 18px;
  font-style: normal;
  font-weight: bold;
  color: ${({ theme }) => theme.colors.darkGrey};
  align-self: flex-start;
`

const StyledDescription = styled.Text`
  font-size: 15px;
  line-height: 22px;
  font-style: normal;
  font-weight: bold;
  color: ${({ theme }) => theme.colors.darkGrey};
`

export default Contacts
