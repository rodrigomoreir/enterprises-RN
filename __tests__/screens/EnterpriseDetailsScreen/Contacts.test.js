import React from 'react'

import {
  renderWithTheme,
  matchSnapshotWithTheme,
} from '../../../src/functions/testing'

import Contacts from '../../../src/screens/EnterpriseDetailsScreen/Contacts'

describe('Contacts', () => {
  test('matches snapshot', () => {
    matchSnapshotWithTheme(
      <Contacts meansOfContact={'TEST'} contact={'test'} />,
    )
  })

  test('renders correctly', () => {
    renderWithTheme(<Contacts meansOfContact={'TEST'} contact={'test'} />)
  })
})
