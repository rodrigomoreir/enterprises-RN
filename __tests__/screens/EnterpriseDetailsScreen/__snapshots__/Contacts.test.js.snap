// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contacts matches snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "flex-end",
        "flexDirection": "row",
      },
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "alignSelf": "flex-start",
          "color": "#696969",
          "fontSize": 18,
          "fontStyle": "normal",
          "fontWeight": "bold",
          "marginTop": 10,
        },
      ]
    }
  >
    TEST
  </Text>
  <Text
    style={
      Array [
        Object {
          "color": "#696969",
          "fontSize": 15,
          "fontStyle": "normal",
          "fontWeight": "bold",
          "lineHeight": 22,
        },
      ]
    }
  >
    test
  </Text>
</View>
`;
